cpp/include/copc-lib/las/header.hpp

```diff
@@ -88,6 +89,8 @@ class LasHeader
     // Returns a box that fits the dimensions of the point cloud based on min and max
     Box Bounds() const;
 
+    void UpdateBounds(const Point &point);
```

Comment:
I think this needs a more clear name, like `CheckAndUpdateBounds`? except not that... idk

Reply:
Hmm yeah I see what you mean, currently could mean you set the bound to the point values... Not sure if I can think of something better than `CheckAndUpdateBounds` though haha.

Reply:
i can't either, so let's roll with it for now 😄